packages/nextjs/app/admin/_components/GrantReview.tsx

```diff
@@ -112,22 +69,38 @@ export const GrantReview = ({ grant }: { grant: GrantDataWithBuilder }) => {
         <BuilderSocials socialLinks={grant.builderData?.socialLinks} />
       </div>
       <p>{grant.description}</p>
-      <div className="flex gap-4 mt-4 justify-end">
+      <div className="flex gap-4 mt-4 justify-between">
         <button
           className={`btn btn-sm btn-error ${isLoading ? "opacity-50" : ""}`}
-          onClick={() => handleReviewGrant(grant, PROPOSAL_STATUS.REJECTED)}
+          onClick={() => handleRejectGrant(PROPOSAL_STATUS.REJECTED)}
           disabled={isLoading}
         >
           Reject
         </button>
-        <button
-          className={`btn btn-sm btn-success ${isLoading ? "opacity-50" : ""}`}
-          onClick={() => handleReviewGrant(grant, acceptStatus)}
-          disabled={isLoading}
-        >
-          {acceptLabel}
-        </button>
+        <div className="flex gap-4">
+          <button
+            className={`btn btn-sm btn-success ${isLoading ? "opacity-50" : ""}`}
+            onClick={() => {
+              if (modalRef.current) modalRef.current.showModal();
+            }}
+            disabled={isLoading}
+          >
+            {acceptLabel}
+          </button>
+          <button
+            className={`btn btn-sm btn-neutral ${isLoading ? "opacity-50" : ""}`}
+            onClick={async () => {
+              const resHash = await sendTx(sendTransactionAsync);
+              // Transactor eats the error, so we need to handle by checking resHash
+              if (resHash && modalRef.current) modalRef.current.showModal();
+            }}
+            disabled={isLoading}
+          >
+            Send 50%
```

Comment:
To me feels more natural to have "Send 50%" button to the left and "Complete/Approve" to the right. Maybe just personal preference!

Reply:
Ohh yeah makes sense !! Update at https://github.com/BuidlGuidl/grants.buidlguidl.com/pull/59/commits/1a0e12ce90ffe1e527a9e0f2756243d0c477c4e0 🙌